packages/consumption/src/modules/attributes/local/LocalAttribute.ts

```diff
@@ -115,7 +115,7 @@ export class LocalAttribute extends CoreSynchronizable implements ILocalAttribut
     }
 
     public isRepositoryAttribute(ownAddress: CoreAddress): this is RepositoryAttribute {
-        return this.isIdentityAttribute() && !this.isShared() && this.isOwnedBy(ownAddress);
+        return this.isIdentityAttribute() && !this.isShared() && this.isOwnedBy(ownAddress) && !this.hasDeletionInfo();
```

Comment:
like below - I think the deletion info doesn't change the type.

Reply:
here also: I think we didn't want to wipe the shareInfo of a shared attribute after deletion, so the isShared check covers this already